### ISSUE TITLE
Accessibility labels

### ISSUE THREAD

**Issue by andreyasko** [unacms/UNA#4377]
Check and add missing aria-labels and alt/role texts for links and buttons:

![image](https://github.com/unaio/una/assets/22210428/d54c31d5-1523-4b96-a95a-4219172f0145)
